GroundControl.gd

```diff
@@ -0,0 +1,67 @@
+extends Node2D
+
+var o2_button = 0
```

Comment:
Let's stick with full word descriptors since that's what you did everywhere else.